### PR TITLE
Email subject added.

### PR DESCRIPTION
**Pull request by draganaHrcek**

### CODE REVIEW

File: core/src/main/java/eu/execom/testgenerator/core/service/email/EmailServiceImpl.java [hunk 1]
```diff
@@ -64,6 +68,7 @@ public void inviteEmployeeAsReviewer(String inviteeEmail, String serverUrl) {
 
     SimpleMailMessage message = new SimpleMailMessage();
     message.setTo(inviteeEmail);
+    message.setSubject("Test generator invitation");
```
Comment (stanislavgrujic) [inline core/src/main/java/eu/execom/testgenerator/core/service/email/EmailServiceImpl.java:71]: Is this subject not read from a template?

File: core/src/main/java/eu/execom/testgenerator/core/service/email/EmailServiceImpl.java [hunk 2]
```diff
@@ -127,6 +132,7 @@ public void inviteApplicant(String applicantEmail, String inviteCode) {
     SimpleMailMessage message = new SimpleMailMessage();
 
     message.setTo(applicantEmail);
+    message.setSubject("Test generator invitation");
```
Comment (stanislavgrujic) [inline core/src/main/java/eu/execom/testgenerator/core/service/email/EmailServiceImpl.java:135]: Please use subject - *You have been invited to take a test in Execom*.

File: core/src/main/java/eu/execom/testgenerator/core/service/email/EmailServiceImpl.java (hunk 1 above)
Comment (stanislavgrujic) [inline core/src/main/java/eu/execom/testgenerator/core/service/email/EmailServiceImpl.java:71]: Please use subject - *You have been invited to review a test*

Comment (draganaHrcek) [reply core/src/main/java/eu/execom/testgenerator/core/service/email/EmailServiceImpl.java]: This service invites employee to become a reviewer. Can I set this title here - "You have been invited to become a reviewer"?  Your subject I will use in another service which assigns reviewer to test. :)

**Pull request merged**